### DISCUSSION TITLE
How to exclude certain app in window mode?

### DISCUSSION THREAD

**Discussion by OriLiMu** [davatorium/rofi#1973] [Q&A]
I want google chrome exclude in window mode. 
I try the command 
`rofi -show combi -show-icons -combi-modes "window,drun,run" -matching regex -filter '^((?!chrome).)*$'` 
Still not work.

another question any way to hide the regex in the search bar? It looks messy with long regex string.

**Comment by DaveDavenport** [comment]
There is currently no way to exclude/hide windows from the window mode.

**Reply by marinjurjevic** [reply]
Is there a way to filter them based on workspace number?
Continuing on this [context](https://github.com/davatorium/rofi/discussions/2156#discussioncomment-13673929), I'll have a need to filter workspaces based on their numbers. E.g. I want to show only windows in range 11-20
